src/design-pages/passkey_authentication.rst

```diff
@@ -0,0 +1,318 @@
+string (similar to pamu2fcfg). Instead of calling the helper utility from the
+command line the system administrator would use 
+`sssctl <https://docs.pagure.org/sssd.sssd/design_pages/sssctl.html>`__. This
+would enable to unify the user experience. Then, the output of this command can
+be set to the LDAP server in a dedicated attribute. For AD we could use the
+`altSecurityIdentities <https://learn.microsoft.com/en-us/windows/win32/adschema/a-altsecurityidentities>`__
+with a prefix (i.e. passkey). For FreeIPA we would use a new attribute called
+ipapasskey from the ipapasskeyuser objectclass. For any other LDAP server we
+would use the passkey attribute.
+
+The format for
+`server-side credentials <https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Resident_Keys.html>`__
+for the key mapping will be
+``passkey:credentialId,pemPublicKey``.
+
+The format for
+`discoverable credentials <https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Resident_Keys.html>`__ for the key mapping will be
+``passkey:credentialId,pemPublicKey,userId``.
+
+IPA process
++++++++++++
+FreeIPA would provide a direct way to register the passkey attributes to the
+user entry. This can be done with a command of IPA CLI or directly in the
+WebUI. The initial implementation would focus on the IPA CLI integration. As
+mentioned before the user would call sssctl to register the key. The output of
+this command would be used to populate the passkey attribute in FreeIPA.
+
+On top of that, FreeIPA can take advantage of libfido2 to communicate with the
+device to perform this whole process.
+
+AD (and any other LDAP server) process
+++++++++++++++++++++++++++++++++++++++
+The users, or the system administrator, would call sssctl. The user can
+manually add this information to their AD object using AD's 'User and
+Computers' utility, or any other attribute in case of another LDAP server.
+
+LDAP attributes
+***************
+Newly added IPA ``ipapasskey`` LDAP attribute will be added to
+``src/providers/ipa/ipa_opts.c`` to retrieve the passkey mapping data
+string as part of the user search in the backend. Store this in the sysdb. The
+passkey mapping is obtained during the registration phase. Example:
+
+* ipapasskey: passkey:9S87qLk8/RxYJ3skwwYduomAM+/HDtz41N0+w/vRL6aGKJkLMsg+2OhO0E8pK5DuO1KmdK61K8PmH7jiYuOqbg==,9YE1s/f7J47h2A/DXCVFWulqoBXFzCSxcbGEBadkpSUFjwUudhPLnPUTv2qNamakXJgRYCZQ7vpS/t5zXMLnkw==
+
+In the Active Directory case, the
+`altSecurityIdentities <https://learn.microsoft.com/en-us/windows/win32/adschema/a-altsecurityidentities>`__
+LDAP attribute will be used to store the passkey mapping data. Example:
+
+* altSecurityIdentities: passkey:9S87qLk8/RxYJ3skwwYduomAM+/HDtz41N0+w/vRL6aGKJkLMsg+2OhO0E8pK5DuO1KmdK61K8PmH7jiYuOqbg==,9YE1s/f7J47h2A/DXCVFWulqoBXFzCSxcbGEBadkpSUFjwUudhPLnPUTv2qNamakXJgRYCZQ7vpS/t5zXMLnkw==
+
+Finally, newly added IPA ``iparequireuserverification`` LDAP attribute will be
```

Comment:
Same here. Please rephrase -- no need for 'newly added ... LDAP attribute', it will not be new by the time it is merged in IPA already, just state the attribute name.